src/libs/AxelarQueryAPI.ts

```diff
@@ -256,8 +255,8 @@ export class AxelarQueryAPI {
     sourceChainId: EvmChain | string,
     destinationChainId: EvmChain | string,
     sourceChainTokenSymbol: GasToken | string,
-    gasLimit: BigNumberish = DEFAULT_ESTIMATED_GAS,
-    gasMultiplier = 1.1,
+    gasLimit: BigNumberish,
+    gasMultiplier = 1.0,
```

Comment:
```suggestion
    gasMultiplier = null,
```
Default should be `null`. Or `"auto"` if we want to make it explicit. In this case, we multiply `execute_gas_multiplier` to the gas price for the execution gas, or the express version of it, if the call is being expressed.

Reply:
oh i just realize that we have `execute_gas_multiplier` returned from the `getFees` API. Then, I agree with you that we should use that as a default value.